GoogleForCreators/web-stories-wp
# Fix video play/pause state after DND

## Summary



## Relevant Technical Choices

Just pushing `null` was not enough, we first want to reset the styles, then communicate that drop is finished with `null`.

## To-do



## User-facing changes



## Testing Instructions

Drop video from the library to canvas background, then deselect/move the mouse out and check if you can pause/play the video (see the controls).

---



Fixes #2618, Fixes https://github.com/google/web-stories-wp/issues/4024
